---
title: Haplotype-resolved genome assembly provides insights into evolutionary history of the tea plant Camellia sinensis
authors: Xingtan Zhang, Shuai Chen, Longqing Shi, Daping Gong, Shengcheng Zhang, Qian Zhao, Dongliang Zhan, Liette Vasseur, Yibin Wang, Jiaxin Yu, Zhenyang Liao, Xindan Xu, Rui Qi, Wenling Wang, Yunran Ma, Pengjie Wang, Naixing Ye, Dongna Ma, Yan Shi, Haifeng Wang, Xiaokai Ma, Xiangrui Kong, Jing Lin, Liufeng Wei, Yaying Ma, Ruoyu Li, Guiping Hu, Haifang He, Lin Zhang, Ray Ming, Gang Wang, Haibao Tang, Minsheng You
year: 2021
---

* Mdnotes File Name: [[zhang2021c]]

# Extracted Annotations (2021-07-17)

> "However, evolutionary consequences of mutation load in clonally propagated crops remain unclear." ([Zhang et al 2021:1](zotero://open-pdf/library/items/3QITB897?page=1))

> "Tea is classified into two varieties, C. sinensis var. sinensis (CSS) and var. assamica (CSA) with a number of distinct features, such as leaf size" ([Zhang et al 2021:1](zotero://open-pdf/library/items/3QITB897?page=1))

> "Nevertheless, the complex evolutionary history and uncertain phylogeny, especially the reticulate evolutionary pattern with wild close relatives, remain to be examined." ([Zhang et al 2021:2](zotero://open-pdf/library/items/3QITB897?page=2))

> "indicating the importance of heterosis in tea breeding14." ([Zhang et al 2021:2](zotero://open-pdf/library/items/3QITB897?page=2))

> "Our results provide insight into the mechanism of heterosis and the evolutionary history of the tea plant and uncover important signatures of selection." ([Zhang et al 2021:2](zotero://open-pdf/library/items/3QITB897?page=2))

> "14,691 of 42,628" ([Zhang et al 2021:2](zotero://open-pdf/library/items/3QITB897?page=2))

> "We next investigated allelic imbalance, that is, allele-specific expression (ASE), without resequencing the parental genomes." ([Zhang et al 2021:4](zotero://open-pdf/library/items/3QITB897?page=4))

> "ASEGs were classified into two major patterns: consistent ASEGs and inconsistent ASEGs" ([Zhang et al 2021:9](zotero://open-pdf/library/items/3QITB897?page=9))

> "Study of the mechanism of widespread ASE possibly due to epigenetic modifications29-32 is a further work that deserves much effort." ([Zhang et al 2021:9](zotero://open-pdf/library/items/3QITB897?page=9))

> "A phased genome based on single sperm sequencing reveals crossover pattern and complex relatedness in tea plants." ([Zhang et al 2021:9](zotero://open-pdf/library/items/3QITB897?page=9))

> "Zhan, D. & Zhang, X. Khaper: a k-mer based haplotype caller (version 1.0). Zenodo https://doi.org/10.5281/zenodo.4780792 (2020)." ([Zhang et al 2021:9](zotero://open-pdf/library/items/3QITB897?page=9))

> "Zhang, X., Zhang, S., Zhao, Q., Ming, R. & Tang, H. Assembly of allele-aware, chromosomal-scale autopolyploid genomes based on Hi-C data. Nat. Plants 5, 833-845 (2019)." ([Zhang et al 2021:9](zotero://open-pdf/library/items/3QITB897?page=9))

> "Patterns of genome-wide allele-specific expression in hybrid rice and the implications on the genetic basis of heterosis. Proc. Natl Acad. Sci. USA 116, 5653-5658 (2019)." ([Zhang et al 2021:10](zotero://open-pdf/library/items/3QITB897?page=10))

> "A total of 190 Camellia accessions were used in the present study, containing 113 CSS, 48 CSA, one C. sinensis var. pubilimba, 15 C. taliensis, 12 closely related species and one C. oleifera as the outgroup." ([Zhang et al 2021:11](zotero://open-pdf/library/items/3QITB897?page=11))

> "with optimized parameters designed for polyploid genomes to assemble heterozygous genome sequences as far as possible" ([Zhang et al 2021:11](zotero://open-pdf/library/items/3QITB897?page=11))

> "For the monoploid genome, we first used our newly developed program Khaper to select primary contigs and filter redundant sequences (Supplementary Note 1) from the initial Canu assembly." ([Zhang et al 2021:11](zotero://open-pdf/library/items/3QITB897?page=11))

> "For the haplotype-resolved genome assembly," ([Zhang et al 2021:11](zotero://open-pdf/library/items/3QITB897?page=11))

> "Finally, two haplotypes (HA and HB) were fully resolved at the chromosomal level." ([Zhang et al 2021:11](zotero://open-pdf/library/items/3QITB897?page=11))



