研究上位性的一个很好的材料。变异（fisher）
找出的位点能够进一步验证。
用两种GWAS方式可取。GWAS角度





这项研究使用了含有550个个体的MAGIC群体用于定位纤维品质相关基因。使用了两种GWAS方法定位，这两种方法互为补充，总共鉴定出39个候选位点。通常由于群体的遗传结构问题，我们很难给出一个最好的GWAS方法，但是每种GWAS方法都为我们提供了一种工具去发现其他方法所忽略的关联。MAGIC群体相较于自然群体，没有群体分层，进而可以控制结果的假阳性。除此之外，MAGIC群体也是用来研究上位性的好材料。上位性的概念最早有 
Bateson在二十世纪初引入，用来描述一个遗传变异的效应被另一个遗传变异掩盖的现象。后来fisher从统计学的角度将其定义为两个位点的共同效应偏离它们的加性组合。上位性是增加表型多样性的一个重要因素，但由于遗传材料和工具的问题，上位性的研究在作物中并不多见。此研究从全基因组的水平探究了棉花纤维品质性状中的上位性问题，发现上位性普遍存在的现象。

This study used a MAGIC population of 550 individuals to map loci related to fiber quality. Two complementary GWAS methods were used and a total of 39 candidate QTL were identified. It's necessary to try different GWAS methods because in general, it is probably not possible to identify a single best GWAS method for all situations due to the biological complexity among GWAS samples, but each GWAS method does provide a tool to uncover the associations that may be missed by other methods. Another advantage of this study is the use of MAGIC population. Compared with natural populations, MAGIC population have no obvious population stratification, which can control false positive results. Also, the MAGIC population is good for studying epistasis. The concept of epistasis was first introduced by Bateson in the early twentieth century to describe the phenomenon where the effect of one genetic variation is masked by another genetic variation. Fisher later defined epistasis as the statistical deviation from the additive combination of two loci in their effects on a phenotype. Although epistasis is an important factor that contribute to the phenotypic diversity, the significance of epistasis is not well understood in crops, due to the limitations of genetic and genomic tools and resources. This study explored the epistasis at the whole genome level, and found that epistasis was prevalent and should be carefully considered during the cotton breeding process. All in all, this study comprehensively analyzed the genetic architecture of cotton fiber quality traits and will facilitate future breeding for fiber quality. My comments are listed below:

1. (Line 95-96) What does "allelic series" mean?
2. In Fig. 6F, I want to know how to caculate the recombination frequency and why you said the epiQTL may experience selection.
3. (Line 310) How do you caculate the phenotypic variations explained by epistatic pairs?
4. (Line 389) Delete "~" in "several ~2-Mb regions", because the sliding window used to count the recombination events is exactly 2 Mb.
5.  The statistical tests should be indicated whether they are one- or two-sided (Line 238, 270, 295, 559, 579, 592)






1. 位点互作对遗传力的贡献如何计算？导入数据时是否将0-1-2这些值转成因子变量吗？
可以先将所有显著标记（以及群体结构等协变量）的加性效应排除掉（regress out），然后剩余残差再计算互作项的解释方差。
2. IBD GWAS如何确定合适的阈值？
3.  互作解释变异比例的问题
4.  标准化加性效应如何定义？
5.  Furthermore, we found the recombination frequency in the epiQTL regions was much lower than the random level (Fig. 6F), suggesting that epiQTL regions may be experiencing selection. 重组率如何计算
6.  为什么两种GWAS方法检测到的位点大部分不重合？
7.  删除389行的~