2021-09-17 Friday

## mashr
nominal表头不需要删除
cd ~/ga/mashr_tensorQTL

- [x] 重新跑
bsub -q smp -o %J.out -e %J.err -n 20 "sh run1.sh"


---
## 文献阅读

### LD score regression analysis
We initially fine-mapped our eQTLs with CAVIAR (Hormozdiari et al., 2014), made maximal posterior inclusion probability annotations (MaxCPP) as previously reported (Hormozdiari et al., 2018) and performed tissue-by-tissue stratified LD score regression (S-LDSC) (Bulik-Sullivan et al., 2015; Finucane et al., 2015) adjusting for functional annotation (“baseline model” provided by the developers).

To evaluate the effect of cell type-shared eQTLs, we made meta-analyzed annotation of 6 representative immune cell subset eQTLs (Mem CD4, Mem CD8, naive B, CL Mono, NK and Neu) using ==linear mixed model based== meta-analysis (Sul et al., 2013). We utilized fixed-effect P values after meta-analysis and fine-mapped eQTLs with CAVIAR. Subsequently, we obtained MaxCPP of meta-analysis annotation, which we call “Meta tissue” annotation. Meta tissue annotation showed significant heritability enrichment for some traits (Figure S4E). To consider tissue-shared eQTL effects together with tissue-specific eQTL effects, we jointly regressed LD scores for each cell eQTL annotation and meta-tissue annotation together with functional annotation.

Formatted GWAS summary statistics for LDSC by developers were downloaded from https://alkesgroup.broadinstitute.org/sumstats_formatted/.



### Fine-mapping of eQTLs
We fine-mapped eQTL variants of GTEx v8 whole blood data of European ancestry and ImmuNexUT classical monocytes and neutrophils. In order to assess loci whose eQTL causal variants were shared between 2 datasets, we limited our analysis to eVariants that were polymorphic in both, eGenes that were significant in both and the eGene whose top eVariant in GTEx had a P value < 1 × 10−5 in ImmuNexUT. These limitations resulted in 2,575 genes for classical monocytes and 2,359 genes in neutrophils. For fine-mapping of GTEx and our eQTLs, we ranked eVariants by P values in GTEx and analyzed the top 50 variants with CAVIAR (Hormozdiari et al., 2014) while permitting up to 5 causal variants. For joint fine-mapping analysis, we utilized eCAVIAR (Hormozdiari et al., 2016) for the same set of variants while permitting up to 5 causal variants. Our whole genome data were utilized for estimating LD for Japanese, and the 1000 genomes project phase 3 European samples were used for estimating LD for Europeans. We compared the number of variants in fine-mapped credible sets that were assumed to include all the causal variants with 95% confidence (95% credible set).

To assess the functional relevance of eQTLs, we utilized 18,734 autosomal variants that had significant impacts on regulatory activity using the survey of regulatory elements (SuRE) reporter technology (van Arensbergen et al., 2019) in K562 cells. We made 10,000 null sets of variants from 5,748,566 autosomal variants that were analyzed in SuRE study while adjusting for the distance to TSS with SuRE significant variants. The enrichment score was calculated by dividing the observed overlapped number of variants by the mean of the number of overlaps in the null sets. Confidence intervals were estimated using 10,000 times bootstrapping of randomly sampled null sets with replacement.


网页翻译助手


