2021-09-13 Monday

## 跑 tensorQTL

- [ ] 生成基因的TSS位置信息

~/cotton_fiber/genome/Garboreum_genome_HAU_v1.0/change.Chr_genome_final_gene_0based_TSS.bed

```shell
cat change.Chr_genome_final_gene_1based.bed |awk '{if ($6=="+") print $1,$2,$4;else print $1,$3,$4}' OFS='\t' > change.Chr_genome_final_gene_1based_TSS.bed
```


- [ ] 准备基因表达bed文件

~/ga/expression/expression_data


```shell
awk 'NR==FNR{a[$4]=$1"\t"$2"\t"$3;next}{$1=a[$1]"\t"$1;print}' ~/cotton_fiber/genome/Garboreum_genome_HAU_v1.0/change.Chr_genome_final_gene_0based_TSS.bed Ga_8DPA_AllSamples_201_FPKM_AllStages_norm.txt > Ga_8DPA_AllSamples_201_FPKM_AllStages_norm.bed
```

### 参考

表达bed文件格式要求

```
#Chr start end ID UNR1 UNR2 UNR3 UNR4
chr1 173863 173864 ENSG123 -0.50 0.82 -0.71 0.83  
chr1 685395 685396 ENSG456 -1.13 1.18 -0.03 0.11  
chr1 700304 700305 ENSG789 -1.18 1.32 -0.36 1.26
```


- Phenotypes must be provided in BED format, with a single header line starting with `#` and the first four columns corresponding to: `chr`, `start`, `end`, `phenotype_id`, with the remaining columns corresponding to samples (the identifiers must match those in the genotype input). The BED file should specify the center of the _cis_-window (usually the TSS), with `start == end-1`. A function for generating a BED template from a gene annotation in GTF format is available in [pyqtl](https://github.com/broadinstitute/pyqtl) (`io.gtf_to_tss_bed`).



## independent eQTL 鉴定
2021年09月14日


R包`qvalue`没装，导致没有q value

`pip install rpy2`

```R
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")

BiocManager::install("qvalue")
```

跑'cis' mode
`KeyError: 'Contig00200'`，有的染色体变异文件中没有

把没有在染色体上的基因删掉。221个。
`zcat Ga_4DPA_AllSamples_202_FPKM_AllStages_norm.bed.gz|grep Contig|wc -l`



任务投递
```
bsub -q gpu -J 4D_GPU -o %J.out -e %J.err "sh indep_GPU.sh"
bsub -q gpu -J 8D_GPU -o %J.out -e %J.err "sh indep_GPU_8D.sh"
bsub -q gpu -J 12D_GPU -o %J.out -e %J.err "sh indep_GPU_12D.sh"
bsub -q gpu -J 16D_GPU -o %J.out -e %J.err "sh indep_GPU_16D.sh"
bsub -q gpu -J 20D_GPU -o %J.out -e %J.err "sh indep_GPU_20D.sh"
```


gpu01满了之后会投递到gpu02。gpu02不可用。 #集群 

24000个基因用GPU跑，大概需要3个小时。

![[Pasted image 20210914113102.png]]

8D 在gpu02节点上，就用不了GPU

```
JOBID      USER    STAT  QUEUE      FROM_HOST   EXEC_HOST   JOB_NAME   SUBMIT_TIME
42185051   zyqi    RUN   q2680v2    login02     sg21        jupyter    Sep 13 15:51
42228731   zyqi    RUN   gpu        login02     gpu01       4D_GPU     Sep 14 11:17
42228826   zyqi    RUN   gpu        login02     gpu02       8D_GPU     Sep 14 11:28
42228931   zyqi    RUN   q2680v2    login02     sg22        *PU_12D.sh Sep 14 11:35
42228942   zyqi    RUN   high       login02     c01n14      16D        Sep 14 11:36
42228953   zyqi    RUN   high       login02     c01n05      20D        Sep 14 11:37
```

23:07
- 12D, 16D GPU只有2个小时不到，~/ga/tensorQTL/run_tensorQTL/GPU
- 16D high 7个小时。12D q2680v2 12个小时都没跑完



## 软件安装
08:38 整理了一下印象笔记，重新命名和移动了一些文件夹

https://github.com/broadinstitute/pyqtl  Python包 qtl 一直在更新

mashr环境安装包
- install.packages("pheatmap")

- pip install bioinfokit # gff转gtf，画图

- python -m ipykernel install --user --name python3_mashr --display-name "Python3 (mashr)" # 将mashr环境下的内核加到jupyter lab中




pip uninstall qtl 不会卸载对应的依赖

plink 转格式 [[plink]]

- [ ] vcf文件添加ID

```
zcat filter2_Q600_SNPs_joint_216.vcf.gz|awk '{if(/^#/){print}else{$3=$1"_"$2;print}}' OFS='\t' > filter2_Q600_SNPs_joint_216_addID.vcf
```


module load HTSlib/1.9
bgzip filter2_Q600_SNPs_joint_216_addID.vcf
tabix filter2_Q600_SNPs_joint_216_addID.vcf.gz

- [ ] 用plink转vcf文件

~/ga/tensorQTL/Genotypes

module load plink
plink --vcf filter2_Q600_SNPs_joint_216_addID.vcf.gz --out filter2_Q600_SNPs_joint_216_addID --allow-no-sex --allow-extra-chr --make-bed
