2021-10-04 Monday

Reviewers' comments:

  

Reviewer #1 (Remarks to the Author):

  

In this study, Wang et al. presented a genomic analysis of a cotton MAGIC population of 550 individuals and identified genetic loci related to fiber quality using two complementary GWAS methods. The merit of this study is to trace the IBD blocks that were used for GWAS analysis. As shown in the manuscript, some novel QTL were found with new candidate genes. The most interesting thing is that the authors performed a genome-wide epistasis study for cotton fiber quality. Although epistasis has long been recognized as an important factor that contributes to phenotypic diversity, its significance is not well understood in cotton, due to the limitations of genetic and genomic tools and resources. This study found that epistasis was prevalent and should be carefully considered during the cotton breeding process. In summary, this study comprehensively analyzed the genetic architecture of fiber quality-related traits and provided a useful resource for future fiber quality improvement. The manuscript is well written, and I only have some minor concerns shown below:

  

1. Line 95-96: What does "allelic series" mean?

  

I learned the term from Liu, H.-J. et al. Genome Biology 21, 20 (2020). It means multiple alleles of a QTL. For example, 4-allelic QTL is expressed as 2 incompletely linked QTN at a given QTL in that paper. This is the phenomenon in which different variations at the same locus lead to the same or very similar phenotypes.

  

2. Line 310: How do you calculate the phenotypic variations explained by epistatic interactions? The method should be specified.

The calculation was very similar to the calculation of phenotypic variations explained by additive effect and the only difference is that we thought about the epistasis effect when we parittioned the observed variance of phenotypes into components attributable to different sources of variation. In more detail, we first regressed out the covariants and the additive effect of significant epistatic pairs and then we used the residuals to calculate the percentage of the variance explained by epistatic interaction. All the calcultions were done in R using lm() function. The method was mentioned in the manuscript, but it may not be clear. I've changed the description of the method and add some details in the manuscript.

3. Fig. 6F, Line 339: I want to know how to calculate the recombination frequency and why you said the epiQTL may experience selection.

Before calculating the recombination frequency, we've used the hidden Markov Model to infer every genomic segment of each offspring descended from any given parent. So we can know whether or not an epistatic pair undergoes recombination in each offspring. The recombination freequency is the ratio of recombined offsprings to all offsprings. The simulated epistatic pairs match the length and number of the real epistatic pairs. We found that epistatic pairs are more likely to maintain the original combination in the offspring, indicating that such a combination is functional and may have undergone selection.

4. Line 389: Delete "~" in "several ~2-Mb regions", because the sliding window used to count the recombination events is exactly 2 Mb.

  
  
  

5. The statistical tests should be indicated whether one- or two-sided was used (Line 238, 270, 295, 559, 579, 592).

two-tailed Student’s t-test

  
  

6. Check the grammar and the sentences to make them correct and more clear. Such as Line 55, there should no the “,” between family and which;

  

Line 56-57: it is better to say “with morphological variations and differed fiber characteristics.”; L59: “which are defined in terms of days post-anthesis (DPA).” Ambiguous description; L64: “Cotton fiber quality traits are quantitative” should be “Cotton fiber quality traits are quantitatively inherited”. And so on……